next.config.js

```diff
@@ -11,7 +11,7 @@
 const withNextIntl = require('next-intl/plugin')()
 
 const nextConfig = {
-    reactStrictMode: true,
```

Comment:
I'm not sure we should get rid of react strict mode

Reply:
Solved